draft-ietf-taps-interface.md

```diff
@@ -1669,12 +1685,55 @@ Type:
 Default:
 : Disabled
 
-This property specifies how long to wait before deciding that a Connection has
-failed when trying to reliably deliver data to the destination. Adjusting this Property
+This property specifies how long to wait before deciding that an active Connection has
+failed when trying to reliably deliver data to the Remote Endpoint. Adjusting this Property
 will only take effect when the underlying stack supports reliability. The special value
 `Disabled` means that this timeout is not scheduled to happen. This can be a valid
 choice with unreliable data transfer (e.g., when UDP is the underlying transport protocol).
+See also {{keep-alive-timeout}}.
+
+### Timeout for keep alive packets {#keep-alive-timeout}
+
+Name:
+: keepaliveTimeout
+
+Type:
+: Numeric, with special value `Disabled`
+
+Default:
+: Disabled
```

Comment:
If it's possible to set this to disable we actually don't need the boolean on/of switch at all. But don't we usually set it to 0 or -1 and say that's a special value (didn't check but should do this uniformly).

Reply:
I agree that this eliminates the need for the on/off switch (unless I'm mising something?). And no, special values 0 or -1 aren't used anymore, they have been replaced with more appropriate types.

Reply:
The other text surrounding uses /when/, however, I';d be happy with /if/.